README.md

```diff
@@ -12,9 +12,10 @@ WPF Template for Revit Add-Ins including wrapped external methods for execution
 ### Build
 
 1. Clone/download this repository and open the `.sln` at the root of the repository with Microsoft Visual Studio.
-2. Re-link references to `RevitAPI.dll` and others which may be missing.
-3. Build the solution - Building the solution will automatically create and copy the add-in files to the folder for Revit 2019.
-4. Open Revit - Upon opening Revit 2019, there should be a tab called "Template" in Revit, with a button to launch the WPF add-in.
+2. `Install-Package DynamicLanguageRuntime -Version 1.1.0`
```

Comment:
Why is this dependency added? I was able to compile and execute successfully without, but want to make sure it doesn't serve another function.